src/avatar.js

```diff
@@ -38,7 +41,7 @@ class AvatarUI extends React.Component {
 
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = { copyMessage: false };
```

Comment:
nit: Bit odd that this value starts as a boolean and changes to a string. Maybe just use `null` for the initial value? Same one line 76 below.